tests/improved-sanity-test/vars/common.yml

```diff
@@ -8,7 +8,9 @@ g_dir2: 'qe2'
 g_hostname: 'myhostname'
 g_pkg: "wget"
 g_rpm_url: 'https://download.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm'
+g_rpm_url_fedora: 'https://kojipkgs.fedoraproject.org//packages/ltrace/0.7.91/22.fc26/x86_64/ltrace-0.7.91-22.fc26.x86_64.rpm'
```

Comment:
nit, only one slash between fedoraproject.org and packages in the url

Will this url always be there or will it be bumped by new versions at some point?

Reply:
Since it's in a "gold" release it'll be around quite a long time...you can still get 
https://koji.fedoraproject.org/koji/buildinfo?buildID=1088
from 11 years ago.

Reply:
(A trickier question is whether the f26 version of ltrace will still install on e.g. F29AH)

Reply:
> (A trickier question is whether the f26 version of ltrace will still install on e.g. F29AH)

Yeah, I was avoiding that question for now.  I welcome any suggestions for `noarch` packages that we could feed into this role.

Reply:
I've been searching but haven't found anything yet.  I am okay with merging this until we find a better package to use for Fedora, CentOS, and RHEL AH.

Reply:
I looked briefly but couldn't find one.  Should be fine, if it breaks we can deal with it later.

Reply:
Errr....`noarch` would be nice, but probably wouldn't solve the version issue.  I guess we are still on the hunt for a 'latest' URL that would deliver a the latest version of an RPM like `epel-release` does.

Reply:
Those URLs are what's in the rpm-md returned by e.g.
```
[fedora]
name=Fedora $releasever - $basearch
failovermethod=priority
#baseurl=http://download.fedoraproject.org/pub/fedora/linux/releases/$releasever/Everything/$basearch/os/
metalink=https://mirrors.fedoraproject.org/metalink?repo=fedora-$releasever&arch=$basearch
```

And really this is what e.g. `rpm-ostree install ltrace` is doing.  I think though this role falls into a gap where we're using `rpm` directly on top of `ostree admin unlock`  (We should definitely have that as an rpm-ostree feature..., it's sort of there with livefs except there it's persistent)

Anyways, eh...I think this is OK as is.

Reply:
(But if for example we ever wanted to use these tests on multi-arch, we'd need to handle substituting `$basearch` the same way libdnf is doing)

Reply:
> I think though this role falls into a gap where we're using rpm directly on top of ostree admin unlock 

Correct, that's the idea of running this particular role in the sanity test.

> (But if for example we ever wanted to use these tests on multi-arch, we'd need to handle substituting $basearch the same way libdnf is doing)

Shoot...that's going to be an issue in the not so distant future.  $DAYJOBBOSS actually just had a conversation with me about making our tests multi-arch aware.